src/core/layout/qgslayoutitempolyline.h

```diff
@@ -185,6 +188,21 @@ class CORE_EXPORT QgsLayoutItemPolyline: public QgsLayoutNodesItem
      */
     double arrowHeadStrokeWidth() const { return mArrowHeadStrokeWidth; }
 
+    /**
```

Comment:
We usually don't do doxygen for overriden methods -- doxygen itself handles inheriting these from the base class docs.

Reply:
Done